src/components/VideoPlayer/index.tsx

```diff
@@ -109,7 +111,6 @@ const VideoPlayer: React.FC<VideoPlayerProps> = ({
       embedMarkup,
       enableAutoplay || autoplay,
       isPlaythrough || playthrough,
-      overrideAspectRatio,
```

Comment:
is there a reason this override is removed?

Reply:
It was only used by one component that had it set to a constant of 0.5625 (16:9) ... I can certainly add it back in, but it appears to be unused ATM

Reply:
I'm working to make sure that videoplayer has feature-parity with video component. thank you https://github.com/WPMedia/engine-theme-sdk/blob/canary/video-component-to-video-player-component-migration.md

Reply:
Just re-established this, will be applied to the player and it will confine the aspects, then my code will read this, so if anyone specifies the aspect the video scaling will basically be de-activated